cpp/src/arrow/ipc/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
 #
-#   http://www.apache.org/licenses/LICENSE-2.0
+# http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Could you revert this needless white space change?

Reply:
Sorry about that but thanks for noticing. PR at https://github.com/apache/arrow/pull/41701.